awx/main/constants.py

```diff
@@ -3,9 +3,8 @@
 
 import re
 
-from django.utils.translation import gettext_lazy as _
-
 from awx.main.models.inventory import InventorySourceOptions
+from django.utils.translation import gettext_lazy as _
```

Comment:
This should probably not be moved. Traditionally, the framework goes before local imports. It'd be better to enforce it with isort, of course.

Reply:
I believe this has been resolved since I removed the unused imports in this commit [e65f098514f1d31064522a57a1fe874cb6f02924](https://github.com/ansible/awx/pull/15537/commits/e65f098514f1d31064522a57a1fe874cb6f02924)

Reply:
Ack. Plz resolve.

Reply:
FWIW, when I mentioned `isort`, I didn't mean applying it selectively to arbitrary modules in the project. It should be applied to the entire project consistently and integrated to enforce that. But that's not something that belongs in this PR, it's rather a future improvement idea.